pyproject.toml

```diff
@@ -1,8 +1,10 @@
 [tool.poetry]
 name = "great_expectations_cloud"
-version = "0.0.3.dev8"
+version = "0.0.3.dev9"
```

Comment:
Do these pre release tags reset each time a minor version goes out?

Reply:
Yes.
You should play around with it locally.

Imagine we are on `0.0.4`, and we choosing between a new `micro` version or a new pre-release version.

1. New micro version (non-pre-release)
```console
$ invoke version-bump  --standard
local:          0.0.4
pypi latest:    0.0.3

  bumping version ... 
new version:    0.0.5

✅ 0.0.5 will be published to pypi on next merge to main
```
2. New pre-release version
```console
invoke version-bump            
local:          0.0.4
pypi latest:    0.0.3

  bumping version ... 
new version:    0.0.4.dev1

✅ 0.0.4.dev1 will be published to pypi on next merge to main
```